### PR TITLE
rationalize some tpm docs

### PR DESCRIPTION
**Pull request by deitch**
Some of the description of storage was split between two docs, some of the docs were not in the keys table, making it incomplete. Rationalized them so one can see in a single place all the keys, and in a separate single place the storage encryption in the proper security doc.

### CODE REVIEW

File: docs/KEYS-AND-CERTS.md [hunk 1]
```diff
@@ -21,16 +21,10 @@ If EVE is running on hardware with a trusted platform module (TPM), then these k
...
-### Storage keys
-
-EVE implements a vault for application content and volumes in /persist/vault/. This is encrypted using standard filesystem encryption (fscrypt for the ext4 file system and native ZFS encryption for the ZFS file system).
-
-But master key for this is managed using EVE using the TPM to seal the keys under the TPM PCRs, and with an encrypted copy of the master key being sent to the controller as a backup, and which the controller will share with the device after successful attestation as specified in [Measured Boot and Remote Attestation](https://wiki.lfedge.org/display/EVE/Measured+Boot+and+Remote+Attestation).
-
-This master storage key is not kept on disk/ssd; only in memory on the running system plus the TPM-sealed one and in the encrypted backup in the controller.
-
-## Controller certificates used by EVE
+## Controller certificates
```
Comment (eriknordmark) [inline docs/KEYS-AND-CERTS.md:27]: Why change the title? EVE doesn't know all the certificates a controller might use; it merely sees a subset of them.

File: docs/KEYS-AND-CERTS.md [hunk 2]
```diff
@@ -21,16 +21,10 @@ If EVE is running on hardware with a trusted platform module (TPM), then these k
...
 | Onboarding cert | Used initially for onboarding | ECC (P-256) | /config/onboard.cert.pem |  [Registration](REGISTRATION.md) |
 | Endorsement cert | Needed for some use of vTPM | Depends on TPM | Generated by TPM | [Identity of EVE](SECURITY.md#identity-of-eves-instance) |
+| Vault Encryption | Lock/unlock vault |  | Sealed to TPM PCRs  | [Encrypted Data Store](SECURITY.md#encrypted-data-store) |
+| Vault Encryption | Backup for upgrades |  | Encrypted and sent to controller  | [Encrypted Data Store](SECURITY.md#encrypted-data-store) |
 
-### Storage keys
-
-EVE implements a vault for application content and volumes in /persist/vault/. This is encrypted using standard filesystem encryption (fscrypt for the ext4 file system and native ZFS encryption for the ZFS file system).
-
-But master key for this is managed using EVE using the TPM to seal the keys under the TPM PCRs, and with an encrypted copy of the master key being sent to the controller as a backup, and which the controller will share with the device after successful attestation as specified in [Measured Boot and Remote Attestation](https://wiki.lfedge.org/display/EVE/Measured+Boot+and+Remote+Attestation).
```
Comment (eriknordmark) [inline docs/KEYS-AND-CERTS.md:29]: Do we have this reference to the measured boot document somewhere else in the docs directory?

Comment (eriknordmark) [reply docs/KEYS-AND-CERTS.md:29]: I see you added it further down.

File: docs/KEYS-AND-CERTS.md (hunk 1 above)
Comment (deitch) [reply docs/KEYS-AND-CERTS.md:27]: Fair enough. I was trying to make it parallel to some of the rest of the titles, but I see how it is confusing. Changing it back.

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**